Classes/common/CloudantSync.h

```diff
@@ -15,7 +15,11 @@
 
 #import <Foundation/Foundation.h>
 
+#ifdef ENCRYPT_DATABASE
+#import "CDTDatastoreManager+EncryptionKey.h"
+#else
 #import "CDTDatastoreManager.h"
+#endif
```

Comment:
Probably worth a small comment explaining this, and where ENCRYPT_DATABASE is set.